server/settings/tools.py

```diff
@@ -35,6 +35,22 @@ class ProductNameProfile(BaseSettingsModel):
     template: str = SettingsField("", title="Template")
 
 
+class FilterCreatorProfile(BaseSettingsModel):
```

Comment:
We should maybe use enum switcher to "Show all" vs. "Filter by identifier".
```python
def creator_profilte_filter_options():
    return [
        {"value": "creator_identifiers", "label": "Filter by identifier"},
        {"value": "show_all", "label": "Show all"},
    ]


class FilterCreatorProfile(BaseSettingsModel):
    """Provide list of allowed Creator identifiers for context"""

    _layout = "expanded"
    host_names: list[str] = SettingsField(
        default_factory=list, title="Host names"
    )
    task_types: list[str] = SettingsField(
        default_factory=list,
        title="Task types",
        enum_resolver=task_types_enum
    )
    task_names: list[str] = SettingsField(
        default_factory=list,
        title="Task names"
    )
    filter_variant: str = SettingsField(
        "creator_identifiers",
        title="Filter variant",
        enum_resolver=creator_profilte_filter_options,
        conditionalEnum = True,
    )
    creator_identifiers: list[str] = SettingsField(
        default_factory=list,
        title="Allowed Identifiers"
    )
```

Reply:
BTW what about this comment, which adds option to not show any create plugins.

Reply:
That is already handled by creating profile without any allowed creators